features/storage/kvstore/tdbstore/TDBStore.h

```diff
@@ -280,6 +287,32 @@ class TDBStore : public KVStore {
     virtual int reserved_data_get(void *reserved_data, size_t reserved_data_buf_size,
                                   size_t *actual_data_size = 0);
 
+    /**
+     * @brief Get the default TDBStore flash start address and size.
+     *
+     * @param[out] start_address    Default TDBStore start address in flash.
+     * @param[out] size             Default TDBStore size.
+     *
+     * @returns MBED_SUCCESS                        Success.
+     *          MBED_ERROR_INITIALIZATION_FAILED    Failed to initialize flash driver.
+     *          MBED_ERROR_MEDIA_FULL               Default TDBStore space overlaps with program memory.
+     */
+    static int get_default_flash_addresses(bd_addr_t *start_address, bd_size_t *size);
```

Comment:
I don't think this is the right place for these new APIs.
`TDBStore` uses just a `BlockDevice`, it does not have to know directly that the block device is going to be `FlashIAP`, and therefore, it does not have to know the flash map either. If we want to keep the class clean, we should not add that dependency here.

Maybe `kv_config.cpp` or `SystemStorage.cpp` would be better place for these.

Reply:
I'll look into moving this. If it doesn't wind up introducing other dependency issues I think kv_config makes more sense because there are some kvstore-specific assumptions in the defaults that this uses, and systemstorage seems to be more generic.

Reply:
Moved into kv_config, rebased on latest master, and pushed. 
With these changes, `mbed test -t ARMC6 -m LPC55S69_S --app-config tools/psa/tfm/mbed_app.json -n *psa-spm_smoke -DUSE_PSA_TEST_PARTITIONS -DUSE_SMOKE_TESTS_PART1 --compile` passes. Thanks.